src/containerapp/azext_containerapp/custom.py

```diff
@@ -1017,6 +1017,33 @@ def delete_containerappsjob(cmd, name, resource_group_name, no_wait=False):
     return containerapp_job_decorator.delete()
 
 
+def suspend_containerappsjob(cmd, name, resource_group_name):
+    raw_parameters = locals()
+    containerapp_job_decorator = ContainerAppJobDecorator(
+        cmd=cmd,
+        client=ContainerAppsJobPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    containerapp_job_decorator.validate_subscription_registered(CONTAINER_APPS_RP)
+
+    return ContainerAppsJobPreviewClient.suspend(cmd=cmd, resource_group_name=resource_group_name, name=name)
```

Comment:
Please use the `containerapp_job_decorator` to execute containerapp_job_decorator.suspend()